### ISSUE TITLE
Read Zally CLI Configuration from File

### ISSUE THREAD

**Issue by netme** [zalando-incubator/zally#263]
Make CLI configurable via `~/.zally` file. 

### Possible values to store: 
1. Zally URL
2. Token
3. ❓ Path for WIP swagger specification

**Comment by mkulak** [comment]
@netme Token?

**Comment by netme** [comment]
@mkulak , you are right, it's too dynamic to move it to config

**Comment by mkulak** [comment]
@netme what use case you have in mind?
If I'm working on some swagger file and I'd like to continuously validate it, I'd rather pass Zally url and path to swagger spec as command line param and then simply re-invoke the same command to validate it again.

**Comment by netme** [comment]
Yeah, we thought to simplify the workflow and call Zally without additional CLI parameters, e.g.

```bash
zally hello/world.yaml
```

**Comment by mkulak** [comment]
As I said I think it's not worth it.

@fmueller what do you think?

**Comment by fmueller** [comment]
Well, it's the typical stups tooling experience at Zalando. There it is done same way: some `.config` files in user directory and all mandatory static parameters are configured to correct values for our environment. We could have same for Zally and url of zally service.
